### PR TITLE
Even-after-Odd in a LinkedList using C++

### PR DESCRIPTION
**Pull request by somya1252**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

### Description

(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

File: C-Plus-Plus/ds/even-after-odd_LinkedList.cpp [hunk 1]
```diff
@@ -0,0 +1,94 @@
+/* We arrange elements in a Linked List such that all even numbers are placed after odd numbers.
```
Comment (HarshCasper) [inline C-Plus-Plus/ds/even-after-odd_LinkedList.cpp:1]: Can you add a bit more information?

File: C-Plus-Plus/ds/even-after-odd_LinkedList.cpp [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+I/P: 5
+     1 2 2 2 1
+O/P: 1 1 2 2 2
+*/
```
Comment (HarshCasper) [inline C-Plus-Plus/ds/even-after-odd_LinkedList.cpp:5]: Keep the sample I/O for the purpose of examples

File: C-Plus-Plus/ds/even-after-odd_LinkedList.cpp [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+/* Sample I/O
+Enter the size of LinkedList: 5
+Enter the elements of the LinkedList: 1 2 2 2 1
+The resulatant LinkedList is: 1 1 2 2 2
+
+Time Complexity:   O(n)
+Space Complexity : O(1)
+
+*/
```
Comment (HarshCasper) [inline C-Plus-Plus/ds/even-after-odd_LinkedList.cpp:94]: Add a new line at the EOF

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
Resolve merge conflicts.

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
Do the changes mentioned

**Review by ankitaggarwal23 — COMMENTED** [review]
Last Reminder, Are you still working on it.

**Review by ankitaggarwal23 — COMMENTED** [review]
Closing this due to inactivity. Thank you for your valuable time.😊

**Pull request closed without merging**